pkg/controller/node/ipam/cloud_cidr_allocator.go

```diff
@@ -49,9 +49,10 @@ type cloudCIDRAllocator struct {
 	client clientset.Interface
 	cloud  *gce.GCECloud
 
-	// Channel that is used to pass updating Nodes with assigned CIDRs to the background
-	// This increases a throughput of CIDR assignment by not blocking on long operations.
-	nodeCIDRUpdateChannel chan nodeAndCIDR
+	// Channel that is used to pass updating Nodes to the background.
+	// This increases a throughput of CIDR assignment by not blocking
+	// on long operations (which can't be done from event handlers).
```

Comment:
IIUC the increase in throughput is because of parallel processing of those nodes (not just because of making the event handler non-blocking), right?

Reply:
The increase in throughput is due to parallelization.

But non-blocking even handlers will speed up other controllers watching nodes (due to using shared informers one slow handler will slow down all controllers).

Reply:
Yes.. Please update the comment so.

Reply:
done